docker/docker-json-config/agent.d/docker.json

```diff
@@ -3,10 +3,9 @@
 // docker container log directory (which must be mounted into the
 // agent's container).
 {
- monitors: [
-   {
-    module: "scalyr_agent.builtin_monitors.docker_monitor",
-    docker_raw_logs: true
```

Comment:
Please pay attention to the removal of this line -- specifically whether this will impact existing customers?